tests/integration/expires.test.ts

```diff
@@ -0,0 +1,74 @@
+import type { SessionData } from '$lib/index.js';
```

Comment:
There seems to be an eslint error
Would you correct it?

![image](https://github.com/user-attachments/assets/5d665e7d-6d68-4ad0-b0be-9a462fc4c96d)

Reply:
Yep. Sorry for those little problems.